README_dependencies.md

```diff
@@ -242,7 +242,10 @@ If you run into difficulty using them, please refer to their documentation,
 linked here:
 
 - [NANA](http://nanapro.org/en-us/documentation/)
-- [SDL](http://wiki.libsdl.org/FrontPage)
+- [SDL](http://wiki.libsdl.org/FrontPage):
+  Please note that the Conan package for SDL can be challenging to build on some platforms.
+  If you need to use SDL, you may find it easier to install using the
+  [official instructions](http://wiki.libsdl.org/Installation) than to use the Conan package.
```

Comment:
I made this change in a separate commit so that it would be easy to revert if necessary. I am aware that not everyone in this project feels the way I do about the Conan SDL package, but I am fed up with trying and failing to build it when everything else is working perfectly. I have seen it take multiple hours to build on Linux, and it often results in a binary that cannot draw to the screen. I think our users should be warned that it is sometimes a headache, and told that they don't have to use it if they don't want to.

Reply:
@ddalcino I appreciate your frustration. I do have SDL working on MacOS/Windows/Linux in the starter project: https://github.com/cpp-best-practices/cpp_starter_project/blob/main/conanfile.txt#L8

I did have to limit the combination of compiler/OS however.

Since boilerplate doesn't even have any SDL code anymore, we can probably just remove all notes to it here.

Reply:
How silly of me; of course. Let's just take it out...